recipe/build.sh

```diff
@@ -1,24 +1,38 @@
 #!/bin/bash
 
-set -exuo pipefail
+# https://xgboost.readthedocs.io/en/latest/build.html
 
-mkdir -p build-target
 
-if [[ ${cuda_compiler_version} != "None" ]]; then
-    export CMAKE_ARGS="-DUSE_CUDA=ON -DUSE_NCCL=ON -DBUILD_WITH_SHARED_NCCL=ON ${CMAKE_ARGS}"
-fi
+export CUDF_ROOT="${PREFIX}"
+export NCCL_ROOT="${PREFIX}"
 
-# Limit number of threads used to avoid hardware oversubscription
-if [[ "${target_platform}" == "linux-aarch64" ]] || [[ "${target_platform}" == "linux-ppc64le" ]]; then
-    export CMAKE_BUILD_PARALLEL_LEVEL=6
-fi
+CUDA_MAJOR=$(echo ${RAPIDS_CUDA_VERSION} | cut -f 1 -d.)
 
-pushd build-target
-cmake ${CMAKE_ARGS} \
-      -GNinja \
-      -DCMAKE_BUILD_TYPE:STRING="Release" \
-      -DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON \
-      -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+GPU_COMPUTE="60;70;75;80;86;90"
+echo "GPU_COMPUTE=$GPU_COMPUTE"
+
+cmake \
```

Comment:
Would keep this part of upstream changes

```suggestion
cmake ${CMAKE_ARGS} \
```